lnbits/extensions/scrub/views_api.py

```diff
@@ -43,9 +40,7 @@ async def api_links(
 
 
 @scrub_ext.get("/api/v1/links/{link_id}", status_code=HTTPStatus.OK)
-async def api_link_retrieve(
-    r: Request, link_id, wallet: WalletTypeInfo = Depends(get_key_type)
-):
+async def api_link_retrieve(link_id, wallet: WalletTypeInfo = Depends(get_key_type)):
```

Comment:
Can you simply delete the argument of this function without breaking it?

Reply:
yes, `r` was not used and its das in dependency injection